ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/RaoUtil.java

```diff
@@ -83,6 +99,8 @@ public static SystematicSensitivityInterface createSystematicSensitivityInterfac
 
             // We may also want to have a different interface for the first run and the successive runs if we do not wish to
             // compute the PTDFs at every iteration.
+        } else if (raoParameters.getObjectiveFunction().doesRequirePtdf()) {
+            builder.withPtdfSensitivities(raoData.getGlskProvider(), raoData.getCnecs());
```

Comment:
this shouldn't be required if we only wish to compute the PTDFs at the start of the RAO

Reply:
removed, it was unnecessary indeed